src/mame/mame.lst

```diff
@@ -32281,8 +32281,9 @@ uchroma68                       // 1980 Motorola evaluation board for the MC6847
 @source:mr/dribling.cpp
 dribling                        // (c) 1983 Model Racing
 driblinga                       // (c) 1983 Model Racing
-driblingbr                      // bootleg
+driblingam                      // bootleg (Automave)
 driblingo                       // (c) 1983 Olympia
+driblingvm                      // bootleg (Videomac)
 
```

Comment:
Is it necessary to rename the file breaking existing setups? Legitimate question, speaking from an app-compat?

Reply:
> Is it necessary to rename the file breaking existing setups? Legitimate question, speaking from an app-compat perspective

The bootlegs are usually defined by the bootlegger, and not by the bootlegger nationality.
IMHO, "Brazil" says nothing about this particular game itself (tagging it as "Brazilian Portuguese" would be different, tho).

Reply:
Short names change from time to time.  That’s just life.